lc0_main.go

```diff
@@ -728,7 +728,7 @@ func acquireLock(dir string, sha string) (lockfile.Lockfile, error) {
 }
 
 func getNetwork(httpClient *http.Client, sha string, keepTime string) (string, error) {
-	dir := "networks"
+	dir := "client_networks"
```

Comment:
I don't have any preferences, just throwing another variants: `.cache`, `cached-networks`, etc.

Reply:
I like `.cache`.

Reply:
I think I prefer not to start with a . - its not like we're trying to hide something in someone's home directory because we don't have a better place for it...

Reply:
What about `client-cache` then? I liked the idea of `.cache` more than `cliect-networks` because it strongly implies (at least to me) that the directory is not for the user. I have heard people complaining that the client deleted their files that were stored in `networks` (case 1 above was not hypothetical) but I see your point.

Reply:
client-cache is good.